source/features/hide-low-quality-comments.tsx

```diff
@@ -37,7 +37,11 @@ function init(): void {
 		lowQualityCount++;
 	}
 
-	for (const commentText of select.all('.comment-body > p:only-child')) {
+	const notLinkedCommentSelector = location.hash.startsWith('#issuecomment-')
+		? `.timeline-comment-group:not(${location.hash}) .comment-body > p:only-child`
+		: '.comment-body > p:only-child';
```

Comment:
This can also be written as something like:

```js
const target = document.querySelector(':target')

for (const comment of comments) {
	if (comment === target) {
		return
	}
}
```

Reply:
Yeah let's go with that filter instead of making the selector extra verbose